### ISSUE TITLE
"When you reveal an answer, the screen reader reads through both what the student would type in and the MathJax equation without separating them."

### ISSUE THREAD

**Issue by christianp** [numbas/Numbas#831]
From the [BCCampus accessibility review](https://bccampus.ca/wp-content/uploads/2020/09/MathTest-Accessibility-Review-AC.docx).

I think the problem is that the value in the input box is read out, and then the MathJax preview. I think the right thing to do is to have some introductory text like "rendering in mathematical notation" or "interpreted as" before the MathJax preview. I think it's clear from visual context how the preview relates to the input box, so this text should be screenreader-only.

**Comment by christianp** [comment]
I've checked with Orca. I had to tab to the preview in order to get it to read it out after typing, but while reading through the whole page the "interpreted as" definitely helped.
